src/main/java/club/minnced/discord/webhook/exception/HttpException.java

```diff
@@ -19,7 +19,20 @@
 import org.jetbrains.annotations.NotNull;
 
 public class HttpException extends RuntimeException {
-    public HttpException(@NotNull String message) {
-        super(message);
+    private final int code;
+    private final String body;
+
+    public HttpException(int code, @NotNull String body) {
+        super("Request returned failure " + code + ": " + body);
```

Comment:
Maybe it makes sense to exclude the body from the message if its otherwise accessible

Reply:
> 
> 
> Isn't the PR you did exactly the same as it was before?
> Sending the message `Request returned failure <code>: <body>`

I excluded body & code from message in http exception, as @MinnDevelopment metioned. If user will log exception, message will remain the same as it was before. But if you catch it you're able to access exception body & code. For example now im able to catch 401 exceptions, when token is incorrect.

Can you tell me what to comment on? I'm quite new in open source and from my perspective there is not much to comment on here, only the constructor has changed and 2 getters have been added.

Reply:
I mean that you comment what the getters will return. Like this https://github.com/MinnDevelopment/discord-webhooks/blob/530d6032d555b48a879dda2e3e6f6e74d4946289/src/main/java/club/minnced/discord/webhook/WebhookClient.java#L133-L142